git_tracker_name_translator.go

```diff
@@ -0,0 +1,14 @@
+package storybranch
+
+import (
+	"regexp"
+	"strconv"
+)
+
+// GetPivotalTrackerTaskID returns a string containing the task ID of the current branch name of the repository.
+func GetPivotalTrackerTaskID(branchName string) int {
+	re := regexp.MustCompile(`[1-9]+$`)
+	taskIdString := re.FindString(branchName)
```

Comment:
When I run `golint`, I get:

> git_tracker_name_translator.go:11:2: var taskIdString should be taskIDString
git_tracker_name_translator.go:12:2: var taskId should be taskID